Configs/BitwardenShared.xcconfig

```diff
@@ -1,4 +1,6 @@
 #include "./Common.xcconfig"
 #include? "./Local.xcconfig"
 
+BITWARDEN_FLAGS = DEBUG_MENU
```

Comment:
🤔  If we have it in Shared, it doesn't need to be in both Debug and Release, and potentially defeats the point of gating it?